addon/components/o-s-s/badge.ts

```diff
@@ -40,10 +51,30 @@ export default class OSSBadge extends Component<OSSBadgeArgs> {
     return SizeDefinition[size as SizeType];
   }
 
+  get skinClass(): string {
+    const skin: SkinType = this.args.skin;
+
+    if (!skin) return '';
```

Comment:
No default skin?

Reply:
I think there might be a default set on the badge class directly